Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/ThreeColoursParams.cs

```diff
@@ -29,22 +29,22 @@ public class ThreeColoursParams : IFormatParams
         /// <summary>
         /// The minimum color
         /// </summary>
-        public Color MinimumColour;
+        public Color _minimumColour;
```

Comment:
Public not private.. I'll say no more, Just undo the changes please.

Reply:
Reverted